python/tank/platform/qt/tank_dialog.ui

```diff
@@ -30,13 +39,24 @@
      </property>
      <widget class="QWidget" name="page_1">
       <property name="styleSheet">
-       <string notr="true"/>
+       <string notr="true">QWidget#page_1 {
+margin: 0px;
+}</string>
```

Comment:
Explicitly defining a margin, fixes the issue, presumably we were inheriting a different value from Nuke before.

Reply:
I explicitly set it to apply to the page_1 widget since I didn't want to have any unintended changes for children of this widget. But maybe I should just let it propagate down?

Reply:
That makes sense to me. You're doing the minimum change required to make if function as it did before.